backend/src/types/index.ts

```diff
@@ -0,0 +1,5 @@
+export interface Book {
```

Comment:
This is an interface that might be used in both the frontend & the backend, I would suggest creating an `interfaces` directory to add these there. You can find an example of what I mean in my [PR](https://github.com/nidosio/nidos/pull/3/files#diff-2c3d0c9a71733132b703349426b0b1b3).

Reply:
I tried to create a shared folder in the root folder of the project, but managed to get nestjs completely tangled so left it in the backend folder for now. But I'll give it a second try!